fontforge/featurefile.c

```diff
@@ -6208,30 +6158,39 @@ return( type==ft_lookup_end || type == ft_feat_end ||
 	    type == ft_lookup_ref );
 }
 
+/* This function copies entries of a given lookup type into the
+ * lookup_next list of the first argument.  A given lookup is
+ * considered to end on encountering one of the types in
+ * fea_FeatItemEndsLookup or NULL, which is returned.
+ *
+ * When called with type==ot_undef the types are not checked
+ * and assumed to match, a condition verified by some of the
+ * variants of fea_ApplyLookupList.
+ *
+ * When called with another type the function adds elements of
+ * that type to lookup_next until either a list-terminating element
+ * or one of a different type is found.
+ */
 static struct feat_item *fea_SetLookupLink(struct feat_item *nested,
 	enum otlookup_type type) {
     struct feat_item *prev = NULL;
     enum otlookup_type found_type;
 
-    while ( nested!=NULL ) {
-	/* Stop when we find something which forces a new lookup */
-	if ( fea_FeatItemEndsLookup(nested->type) )
-    break;
-	if ( nested->ticked ) {
-	    nested = nested->next;
-    continue;
-	}
-	found_type = fea_LookupTypeFromItem(nested);
-	if ( type==ot_undef || found_type == ot_undef || found_type == type ) {
-	    if ( nested->type!=ft_ap || nested->u2.ap->type!=at_mark )
-		nested->ticked = true;		/* Marks might get used in more than one lookup */
-	    if ( prev!=NULL )
-		prev->lookup_next = nested;
-	    prev = nested;
+    while ( nested!=NULL && !fea_FeatItemEndsLookup(nested->type) ) {
```

Comment:
No logical change here (moving the break condition into the while condition), right?

Reply:
Correct. I found the other syntax to be less easily understood, so as long as I was "in here" I figured I would change it to this.